### ISSUE TITLE
MySQL - Add support to set additional system variables (via configuration)

### ISSUE THREAD

**Issue by nbaua** [typeorm/typeorm#6586]
**Issue type:**

[ ] bug report
[x] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[x ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x ] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

Hi,
There should be a way to set driver level variables through additional configuration options when the connection is started.
I am bit sure that mysql native drivers support passing additional commands while booting up the server connection.

**Problem Statement** 
Basically one of my ViewEntity is based on the GroupConcat functionality, this worked well while the generated data was less than 1024 characters. Still this is not a problem as it seems as **mysql** native drivers has provision to override this configuration as under:

```
SET GLOBAL group_concat_max_len=desired_character_length;
SET SESSION group_concat_max_len=desired_character_length;
```
Now the concern is how to set this variables through node + TypeORM?

It would be a great feature, if some additional configuration mechanism is provided. Thanks.

N Baua
